src/com/owncloud/android/ui/activity/Preferences.java

```diff
@@ -514,6 +546,8 @@ protected void onResume() {
                 PreferenceManager.getDefaultSharedPreferences(getApplicationContext());
         boolean state = appPrefs.getBoolean(PassCodeActivity.PREFERENCE_SET_PASSCODE, false);
```

Comment:
Please rename this variable as passCodeState, to distinguish it better from patternState